src/app/application.cpp

```diff
@@ -665,8 +665,8 @@ int Application::exec(const QStringList &params)
     if (pref->getWebUIPassword() == "ARQ77eY1NUZaQsuDHbIMCA==:0WMRkYTUWVT9wVvdDtHAjU9b3b7uB8NR1Gur2hmQCvCDpm39Q+PsJRJPaCU51dEiz+dTzh8qbPsL8WkFljQYFQ==")
     {
         const QString warning = tr("The Web UI administrator username is: %1").arg(pref->getWebUiUsername()) + '\n'
-            + tr("The Web UI administrator password is still the default one: %1").arg("adminadmin") + '\n'
-            + tr("This is a security risk, please consider changing your password from program preferences.") + '\n';
+            + tr("The Web UI administrator password has still not been changed from the default: %1").arg("adminadmin") + '\n'
```

Comment:
I think "still" is no longer needed. Either way i approve of the PR (in case it takes me a while to officially mark it as such).

Reply:
Done.